R/build_lookup.R

```diff
@@ -3,13 +3,22 @@
 #' @param infile A string, the path to FASTA file.
 #'
 #' @return A character vector of gene names.
+#' @export
+#' @keywords internal
+#' @examples
+#' # Given a FASTA file containing this header:
+#' #   >SomeText|TRBV29/OR9-2*01|MoreText|
+#' #   >SomeText|TRBVA/OR9-2*01|MoreText|
+#'
+#' fasta <- get_example_path("fasta_dir/test_trbv.fa")
+#' parse_imgt_fasta(fasta)
 parse_imgt_fasta <- function(infile) {
   lines <- readLines(infile)
   
   # Extract gene names from headers
   imgt_list <- sapply(lines[grep("^>", lines)], function(line) {
```

Comment:
I would suggest avoid `sapply` and using `vapply` or something else

Reply:
Fixed in 64b0f82d072005247fc31373c9bdbdb59ed4619f